### ISSUE TITLE
Package request: Lampray

### ISSUE THREAD

**Issue by dolerich-hirnfiedler** [NixOS/nixpkgs#304065]
**Project description**
Lampray is a mod manager for gaming on Linux
Currently supported
 Note: At this time, mods using .rar files are not supported.
 - Baldur's Gate 3
 - Cyberpunk 2077

Planned support
 - The Elder Scrolls V: Skyrim (SE and AE)
 - Elder Scrolls IV: Oblivion
 - Elder Scrolls III: Morrowind
 - Fallout 4
 - The Sims 4

**Metadata**

* homepage URL:
* source URL: https://github.com/CHollingworth/Lampray
* license: The Unlicense
* platforms:  linux

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by azahi** [comment]
I don't think that Lampray in its current state is suitable for packaging in Nixpkgs: the repo contains [pre-compiled third-party libraries](https://github.com/CHollingworth/Lampray/tree/master/third-party/bit7z/lib/x64) and other oddities, not to mention the code quality is questionable at best.

In the meantime, I wrote a [derivation](https://git.azahi.cc/nixfiles/tree/packages/lampray.nix) for it, feel free to use it as you see fit.
